SDWebImage/SDWebImageDownloaderOperation.h

```diff
@@ -66,6 +66,8 @@ FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadFinishNotification
  */
 @property (nonatomic, strong, nullable) NSURLCredential *credential;
 
+@property (nonatomic, copy, nullable) Class internalCoderClass;
```

Comment:
This is really a pain. In fact I need to transfer the coder class infomation to the operation from `SDWebImageDownloader`. But I found that our classes are all public and open to user now. There is no what like `UIKitInternal` or `SDWebImageDownloaderOperation+Private` here to keep only visible in project. If you'd like, I can create `SDWebImageDownloaderOperation+Private` and keep to private and removed from source files in podspec.